packages/components/time-picker/src/common/props.ts

```diff
@@ -6,6 +6,9 @@ import type { Options } from '@popperjs/core'
 import type { ComponentSize } from '@element-plus/utils/types'
 
 export const timePickerDefaultProps = {
+  id: {
```

Comment:
We are **not** defaulting the id to empty string (`default: ''`) on purpose. We want to only add the id if we specify a value. Having an empty string id as the default will add additional noise to the html output `<input id="" .... />` and will break existing test snapshots(not truly backwards compatible).
I was also tempted to remove the `name` prop's `default: ''` for the same reason (we have empty name attributes by default `<input name="" ... />`) but didn't do it because it will break snapshots as well.